### PR TITLE
Sc 102

### PR DESCRIPTION
**Pull request by mahir0837**

### CODE REVIEW

File: src/com/company/DataGenerator.java [hunk 1]
```diff
@@ -29,9 +29,13 @@ public static void createCustomer(){
 
         Customer customer1 = new Customer(UUID.randomUUID(),"ozzy","ozzy@cydeo.com",customer1AddressList);
         Customer customer2 = new Customer(UUID.randomUUID(),"mike","mike@gmail.com");
+        Customer customer3=new Customer(UUID.randomUUID(),"Kicchi","kicchi@gmail.com");
+
```
Comment (ZehraSarigedikDemircioglu) [inline src/com/company/DataGenerator.java:33]: Can you please remove empty line?

File: src/com/company/DataGenerator.java [hunk 2]
```diff
@@ -29,9 +29,13 @@ public static void createCustomer(){
...
 
         StaticConstants.CUSTOMER_LIST.add(customer1);
         StaticConstants.CUSTOMER_LIST.add(customer2);
+        StaticConstants.CUSTOMER_LIST.add(customer3);
+
```
Comment (ZehraSarigedikDemircioglu) [inline src/com/company/DataGenerator.java]: Can you please remove empty line?

File: src/com/company/Main.java [hunk 3]
```diff
@@ -9,293 +9,331 @@
...
+import java.util.*;
+import java.util.stream.Collectors;
 
 import static com.company.StaticConstants.DISCOUNT_LIST;
 import static com.company.StaticConstants.ORDER_LIST;
 
 public class Main {
 
-  public static void main(String[] args) {
+    public static void main(String[] args) {
```
Comment (ZehraSarigedikDemircioglu) [inline src/com/company/Main.java]: Can you please reformat this file?

**Review by ZehraSarigedikDemircioglu — CHANGES_REQUESTED** [review]
Can you please update your branch with your fresh master?

File: src/com/company/DataGenerator.java (hunk 1 above)
Comment (mahir0837) [reply src/com/company/DataGenerator.java:33]: I removed empty line. Could you review my pull request please?

**Review by ihccik — APPROVED** [review]
Perfect

**Comment by ZehraSarigedikDemircioglu** [comment]
Great

**Review by ZehraSarigedikDemircioglu — APPROVED** [review]
Great

**Pull request merged**